jade/distribution.yaml

```diff
@@ -2944,6 +2944,23 @@ repositories:
       url: https://github.com/ros-naoqi/naoqi_dashboard.git
       version: master
     status: maintained
+  naoqi_dcm_driver:
+    doc:
+      type: git
+      url: https://github.com/ros-naoqi/naoqi_dcm_driver.git
+      version: master
+    release:
+      packages:
+      - naoqi_dcm_driver
```

Comment:
For a repo containing only a single package with the (now) same name as the repo you have to remove the `packages` list.